spec/plugin_spec_helper.rb

```diff
@@ -137,11 +137,11 @@ def create_non_member_role
       end
 
       def create_anonymous_role
-        create_builtin_role "Anonymous", Role::BUILTIN_ANONYMOUS
+        create_builtin_role 'Anonymous', Role::BUILTIN_ANONYMOUS
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.